.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+# Description of the PR
+
++
+
+
+# Testing
+| unit                                                                                   |  integration                                                                                         |  manual                                             |  
+|----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|-----------------------------------------------------|
+|<ul><li>- [ ] new **unit** tests created</li><li>- [ ] all **unit** tests pass</li></ul>|<ul><li>- [ ] new **integration** tests created</li><li>- [ ] all **integration** tests pass</li></ul>|<ul><li>- [ ] **manual** testsing went well</li></ul>|
```

Comment:
Some newlines here?

Reply:
looked better when I've been creating it... :open_mouth: , but I've added some improvements there))